orc8r/cloud/go/services/ctraced/obsidian/models/swagger.v1.yml

```diff
@@ -0,0 +1,184 @@
+  /networks/{network_id}/tracing/{trace_id}:
+    get:
+      summary: Get tracing status
+      tags:
+        - Call Tracing
+      parameters:
+        - $ref: './orc8r-swagger-common.yml#/parameters/network_id'
+        - $ref: '#/parameters/trace_id'
+      responses:
+        '200':
+          description: Show tracing status
+          schema:
+            $ref: '#/definitions/call_trace'
+        default:
+          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
+    put:
+      summary: Update a call trace
+      tags:
+        - Call Tracing
+      parameters:
+        - $ref: './orc8r-swagger-common.yml#/parameters/network_id'
+        - $ref: '#/parameters/trace_id'
+        - in: body
+          name: Call Trace Configuration
+          description: Call trace
+          required: true
+          schema:
+            $ref: '#/definitions/mutable_call_trace'
+      responses:
+        '204':
+          description: Success
+        default:
+          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
+    delete:
+      summary: Delete a call trace record
+      tags:
+        - Call Tracing
+      parameters:
+        - $ref: './orc8r-swagger-common.yml#/parameters/network_id'
+        - $ref: '#/parameters/trace_id'
+      responses:
+        '204':
+          description: Success
+        default:
+          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
+
+parameters:
+  trace_id:
+    description: Unique ID of call trace
+    in: path
+    name: trace_id
+    required: true
+    type: string
+
+definitions:
+  call_trace:
+    description: Mutable Call Trace
+    type: object
+    required:
+      - trace_id
+      - config
+    properties:
+      trace_id:
+        type: string
+        x-nullable: false
+        example: "pair1"
+      config:
+        $ref: '#/definitions/call_trace_config'
+      state:
+        $ref: '#/definitions/call_trace_state'
+
+  mutable_call_trace:
+    description: Mutable Call Trace
+    type: object
+    required:
+      - trace_id
+      - requested_end
+    properties:
+      trace_id:
+        type: string
+        x-nullable: false
+        example: "pair1"
+      requested_end:
+        type: boolean
+
+  call_trace_config:
+    type: object
+    description: Call Trace spec
+    required:
+      - trace_type
+    properties:
+      trace_type:
+        type: string
+        x-nullable: false
+        enum:
+          - 'GATEWAY'
+          - 'SUBSCRIBER'
+          - 'PROTOCOL'
+          - 'INTERFACE'
+      imsi:
+        type: string
+      protocol:
+        type: string
+      interface:
+        type: string
+      timeout:
+        type: integer
+        format: uint32
+        description: Timeout of call trace in seconds
+
+  call_trace_state:
+    type: object
+    description: Full state object of a call trace
+    properties:
+      call_trace_available:
+        type: boolean
+      call_trace_ending:
```

Comment:
Set to true if requesting a call trace to end. Just weirdness since it's a REST API, but it should pretty much always be true if this is getting called